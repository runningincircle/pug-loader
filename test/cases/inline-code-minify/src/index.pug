doctype html
html(lang="en")
  head
    script(async src='https://www.googletagmanager.com/gtag/js?id=AW-xxxxxxxxx')
    script: :minify
      window.dataLayer = window.dataLayer || [];
      function gtag() {
        dataLayer.push(arguments);
      }
      gtag('js', new Date());
      gtag('config', 'AW-xxxxxxxxx');

  body
    h1 Hello World!