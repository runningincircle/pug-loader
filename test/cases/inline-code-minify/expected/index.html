<!DOCTYPE html><html lang="en"><head><script async src="https://www.googletagmanager.com/gtag/js?id=AW-xxxxxxxxx"></script><script>window.dataLayer = window.dataLayer || [];function gtag() {  dataLayer.push(arguments);}gtag('js', new Date());gtag('config', 'AW-xxxxxxxxx');</script></head><body><h1>Hello World!</h1></body></html>